creator-node/src/routes/prometheusMetricsRoutes.js

```diff
@@ -0,0 +1,12 @@
+module.exports = function (app) {
+  app.get('/prometheus_metrics', async (req, res) => {
+    const prometheusRegistry = req.app.get('serviceRegistry').prometheusRegistry
+
+    try {
+      res.setHeader('Content-Type', prometheusRegistry.register.contentType)
+      res.send(await prometheusRegistry.getAllMetricData())
```

Comment:
Export all metrics, not just the current registry's metrics using `await Prometheus.register.metrics()`:

* https://github.com/siimon/prom-client#register